src/containers/TaskRuns/TaskRuns.js

```diff
@@ -187,10 +181,23 @@ TaskRunsContainer.propTypes = {
 };
 
 /* istanbul ignore next */
-function mapStateToProps(state) {
+function mapStateToProps(state, props) {
   return {
-    namespace: getSelectedNamespace(state)
+    error: getTaskRunsErrorMessage(state),
+    taskRunsByTaskName: getTaskRunsByTaskName(
```

Comment:
can just call this `taskRuns` here since it gets renamed to that when used anyway